coderd/database/migrations/000230_marked_for_deletion_notification_template.up sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO
+    notification_templates (
+        id,
+        name,
+        title_template,
+        body_template,
+        "group",
+        actions
+    )
+VALUES (
+        '51ce2fdf-c9ca-4be1-8d70-628674f9bc42',
+        'Workspace Marked for Deletion',
+        E'Workspace "{{.Labels.name}}" marked for deletion',
+        E'Hi {{.UserName}}\n\n' || E'Your workspace **{{.Labels.name}}** has been marked for **deletion** after {{.Labels.dormancyHours}} hours of dormancy.\n' || E'The specified reason was "**{{.Labels.reason}} (initiated by: {{ .Labels.initiator }}){{end}}**\n\n' || E'Dormancy refers to a workspace being unused for a defined length of time, and after it exceeds {{.Labels.dormancyHours}} hours of dormancy it will be deleted.\n' || E'To prevent your workspace from being deleted, simply use it as normal.',
```

Comment:
Nit: I'm not sure `(initiated by: autobuild)` is going to be useful to folks.
I think we can drop it from the messaging. Initiator is useful in the `workspace deleted` notification because it can be initiated by a human, but this isn't so it won't help the receiver.

Reply:
+1 to drop this